frontend/src/components/Summary/ImplementationRow.tsx

```diff
@@ -51,26 +57,45 @@ const ImplementationRow = ({
         </small>
       </td>
 
-      <td className="text-center align-middle">
-        {implementationResults.totals.erroredCases}
-      </td>
-      <td className="text-center align-middle">
-        {implementationResults.totals.skippedTests}
-      </td>
-      <td className="text-center align-middle details-required">
-        {implementationResults.totals.failedTests! +
-          implementationResults.totals.erroredTests!}
-        <div className="hover-details text-center">
-          <p>
-            <b>failed</b>: &nbsp;{implementationResults.totals.failedTests}
-          </p>
-          <p>
-            <b>errored</b>: &nbsp;{implementationResults.totals.erroredTests}
-          </p>
-        </div>
-      </td>
+      <OverlayTrigger
+        placement="left-end"
+        overlay={
+          <Popover style={{ border: "1px solid var(--bs-primary)" }}>
+            <Popover.Body>
+              <Container className="p-0">
+                <Row className="d-flex flex-column gap-2">
+                  <Col>
+                    <b style={{ fontSize: "1rem" }}>failed:</b>&nbsp;
+                    <span style={{ fontWeight: "500" }}>
```

Comment:
Any reason you're not using the Bootstrap classes for font sizing and weights? https://getbootstrap.com/docs/5.3/utilities/text/#font-weight-and-italics

Reply:
I wanted something that was just slightly less bolder than `fw-bold` and nothing in those options on the docs were satisfying the requirements so I made it 500.

Reply:
It would've been easier if there was a `fw-semi-bold` instead. So had to do 500.